python-sdk/src/astro/sql/operators/export_file.py

```diff
@@ -62,6 +73,77 @@ def execute(self, context: Context) -> File:
         else:
             raise FileExistsError(f"{self.output_file.path} file already exists.")
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Returns the lineage data
+        """
+        input_dataset: list[OpenlineageDataset] = [
+            OpenlineageDataset(
+                namespace=None,
+                name=None,
+                facets={},
+            )
+        ]
+        if isinstance(self.input_data, BaseTable):
```

Comment:
what happens in case of ```self.input_data``` is a dataframe?

Reply:
Look like it would be empty but I feel having {data_type: df, df_size: xyz} might be helpful WDYT?

Reply:
yes, i had a similar question but a dataframe doesn't qualify as a dataset to have a name or URI. That's why the dataframe is kept empty until we come up with something logical.
Sharing open lineage naming for datasets https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md